rust/derive/src/stringenum.rs

```diff
@@ -0,0 +1,96 @@
+/* Copyright (C) 2023 Open Information Security Foundation
```

Comment:
nit: 2024?

Reply:
I started this in 2023 :-p

Reply:
Ah, I always think we use the year of when things get merged. TIL.